src/Test/MakerTestEnvironment.php

```diff
@@ -350,13 +350,20 @@ private function buildFlexSkeleton()
         // fetch a few packages needed for testing
         MakerTestProcess::create('composer require phpunit browser-kit symfony/css-selector --prefer-dist --no-progress --no-suggest', $this->flexPath)
                         ->run();
+        $this->fs->remove($this->flexPath.'/vendor/symfony/phpunit-bridge');
+
+        if ('\\' !== \DIRECTORY_SEPARATOR) {
+            $this->fs->symlink('../../../../../../vendor/symfony/phpunit-bridge', './vendor/symfony/phpunit-bridge');
+        } else {
+            $this->fs->mirror(\dirname(__DIR__, 2).'/vendor/symfony/phpunit-bridge', $this->flexPath.'/vendor/symfony/phpunit-bridge');
+        }
```

Comment:
Using the required phpunit bridge was now a problem?

Reply:
The bridge used for the main test suite and for functional tests must be at exactly the same because they share the same runtime code. This leads to a fatal error right now when main is 4.4 or 5.0 but functional stays at 4.3.